src/frontend/src/components/OnboardingBlock.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box, Grid, Typography, useTheme } from '@mui/material';
+import EditIcon from '@mui/icons-material/Edit';
+import { Organization } from 'shared';
+
+interface OnboardingBlockProps {
+  organization: Organization;
+}
+
+const OnboardingBlock: React.FC<OnboardingBlockProps> = ({ organization }) => {
```

Comment:
Instead of making a new file for this, i think we can just keep it all in the onboardingInfoSection file just to keep it as simple as possible.